src/components/LoginForm/LoginForm.tsx

```diff
@@ -1,18 +1,25 @@
 import { yupResolver } from '@hookform/resolvers/yup';
 import { useState } from 'react';
 import { SubmitHandler, useForm } from 'react-hook-form';
+import toast from 'react-hot-toast';
+import { useNavigate } from 'react-router';
 import Button from '../../components/Button';
 import Icon from '../../components/Icon';
-import Input from '../Input';
 import { useAppDispatch } from '../../hooks/redux';
+import { ROUTES } from '../../routing/routes';
 import { loginSchema } from '../../schemas';
 import { login } from '../../store/user/thunks';
 import { UserCredentials } from '../../types';
+import Input from '../Input';
+import Spinner from '../Spinner';
 import styles from './LoginForm.module.css';
 
 const LoginForm = () => {
   const [shouldShowPassword, setShouldShowPassword] = useState(false);
+  const [isSubmitting, setIsSubmitting] = useState(false);
```

Comment:
It seems `isSubmitting` not clear name. Looks like `isLoading`.
But we have loading state in redux store and cant get it from there